tests/automation_framework/data/work_order/work_order_submit.yaml

```diff
@@ -32,93 +33,53 @@ Default:
     workerId: ''
     workloadId: heart-disease-eval
 test_workordersubmit_OutDataDataEncryptionKey_hyphen:
-  params: '{
-
-    "outData": [
-
-    {"index" : 0,
-
-    "data": "",
-
-    "encryptedDataEncryptionKey": "-",
-
-    "iv": ""
-
-    }
-
-    ]}'
+  params:
+    outData:
+    - data: ''
+      encryptedDataEncryptionKey: '-'
+      index: 0
+      iv: ''
 test_workordersubmit_data_differentdataheartdisease:
-  inData: '[
-
-    {"index": 2,
-
-    "dataHash": "",
-
-    "data": "Heart disease evaluation data: 25 10 1 67  102 125 1 95 5 10 1 11 36
-    1",
-
-    "encryptedDataEncryptionKey": "",
-
-    "iv": ""
-
-    },
-
-    {"index": 1,
-
-    "dataHash": "",
-
-    "data": "Heart disease evaluation data: 32 1 1 156  132 125 1 95  1 0 1 1 3 1
-    2 3 4 -1",
-
-    "encryptedDataEncryptionKey": "",
-
-    "iv": ""
-
-    }]'
+  inData:
+  - data: 'Heart disease evaluation data: 25 10 1 67  102 125 1 95 5 10 1 11 36 1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 2
+    iv: ''
+  - data: 'Heart disease evaluation data: 32 1 1 156  132 125 1 95  1 0 1 1 3 1 2
+      3 4 -1'
+    dataHash: ''
+    encryptedDataEncryptionKey: ''
+    index: 1
+    iv: ''
 test_workordersubmit_data_multipleechoresult:
-  '"inData"': '[
-
-    {"index": 1,
-
-    "dataHash": "",
-
-    "data": "java",
-
-    "encryptedDataEncryptionKey": "",
-
-    "iv": ""
-
-    },
-
-    {"index": 1,
-
-    "dataHash": "",
-
-    "data": "python",
-
-    "encryptedDataEncryptionKey": "",
-
-    "iv": ""
-
-    },
-
-    {"index": 1,
-
-    "dataHash": "",
-
-    "data": "perl",
-
-    "encryptedDataEncryptionKey": "",
-
-    "iv": ""
-
-    }]'
+  '"inData"':
```

Comment:
This keyword is still under single quotes and double quotes

Reply:
Fixed